neo/core/epoch.py

```diff
@@ -227,10 +227,9 @@ def __repr__(self):
         """
 
         objs = [
-            "%s@%s for %s" % (label, str(time), str(dur))
-            for label, time, dur in zip(self.labels, self.times, self.durations)
+            f"{label}@{str(time)} for {str(dur)}" for label, time, dur in zip(self.labels, self.times, self.durations)
         ]
-        return "<Epoch: %s>" % ", ".join(objs)
+        return f"<Epoch: %s>" % ", ".join(objs)
```

Comment:
```suggestion
        return "<Epoch: %s>" % ", ".join(objs)
```

leaving this one as % because I don't want to mess up this join dynamic